nextflow.config

```diff
@@ -37,14 +37,12 @@ params {
     // AnnotSV options
     annotsv_candidate_genes         = null
     annotsv_gene_transcripts        = null
-    annotsv_file_name               = "annotsv_annotated"
 
     // VEP options
     vep_assembly                    = "GRCh38"
     vep_cache_version               = 108
     vep_version                     = 109.3
```

Comment:
you want to keep this and not move to a more recent version? Currently 112

Reply:
Done!